torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -4,7 +4,6 @@
 
 #include <torch/csrc/jit/codegen/cuda/fusion.h>
 #include <torch/csrc/jit/codegen/cuda/ir_base_nodes.h>
-#include <torch/csrc/jit/codegen/cuda/ir_interface_nodes.h>
```

Comment:
Just curious, was this causing any problem?

Reply:
No, it worked fine, though I don't know why. It looks weird to me that two headers include each other.